### PR TITLE
Rename repository

### PR DESCRIPTION
**Pull request by squeed**
Change github.com/openvswitch to github.com/ovn-org.

### CODE REVIEW

File: contrib/roles/linux/openvswitch/tasks/download_install_packages.yml [hunk 1]
```diff
@@ -70,7 +70,7 @@
     line: |
       [Unit]
       Description=openvswitch
-      Documentation=https://github.com/openvswitch/ovs
+      :cn
```
Comment (alinbalutoiu) [inline contrib/roles/linux/openvswitch/tasks/download_install_packages.yml]: I guess this is a typo.

File: go-controller/pkg/config/config.go [hunk 2]
```diff
@@ -830,7 +830,7 @@ func (a *OvnAuthConfig) ensureCACert() error {
 	// SB and NB since ovn-sbctl only supports --bootstrap-ca-cert from
 	// 2.9.90+.
 	// FIXME: change back to a.ctlCmd when sbctl supports --bootstrap-ca-cert
-	// https://github.com/openvswitch/ovs/pull/226
+	// https://github.com/ovn-org/ovs/pull/226
```
Comment (alinbalutoiu) [inline go-controller/pkg/config/config.go]: This shouldn't be changed.

**Comment by aserdean** [comment]
@squeed I sent a PR that updates the `contrib` folder to use the new links. The relevant commit can be found here: https://github.com/ovn-org/ovn-kubernetes/pull/701/commits/04d5c72eda44ef8acce9aacc61fa157fed30b7d7 .

Do you mind punting changing the links from the `contrib` folder in this PR?

File: contrib/roles/linux/openvswitch/tasks/download_install_packages.yml (hunk 1 above)
Comment (squeed) [reply contrib/roles/linux/openvswitch/tasks/download_install_packages.yml]: good catch, thanks

File: go-controller/pkg/config/config.go (hunk 2 above)
Comment (squeed) [reply go-controller/pkg/config/config.go]: yup.

**Comment by squeed** [comment]
@aserdean Thanks for the review. I removed the changes to `contrib`.

**Comment by shettyg** [comment]
@squeed 

Can you please write a Signed-off-by in a message here. The commit missed it.

**Pull request merged**